WeakAuras/Types.lua

```diff
@@ -3222,6 +3222,7 @@ LSM:Register("statusbar", "Stripes", [[Interface\AddOns\WeakAuras\Media\Textures
 LSM:Register("statusbar", "Thick Stripes", [[Interface\AddOns\WeakAuras\Media\Textures\Statusbar_Stripes_Thick]])
 LSM:Register("statusbar", "Thin Stripes", [[Interface\AddOns\WeakAuras\Media\Textures\Statusbar_Stripes_Thin]])
 LSM:Register("border", "Drop Shadow", [[Interface\AddOns\WeakAuras\Media\Textures\Border_DropShadow]])
+LSM:Register("border", "Stealable Glow", [[Interface\TargetingFrame\UI-TargetingFrame-Stealable]])
```

Comment:
this doesn't appear to work. That is the resulting border looks wrong.

Reply:
ho right, i added it as a test and forget to remove later